tmexp/preprocess.py

```diff
@@ -195,11 +183,11 @@ def preprocess(
             for feature, feature_word_dict in word_dict.items()
         }
         lang_count[lang] += 1
-    logger.info("Parsed {} distinct files:".format(sum(lang_count.values())))
+    logger.info("Parsed %d distinct files:" % sum(lang_count.values()))
     for lang in sorted(lang_count):
-        logger.info("   {} : {} files.".format(lang, lang_count[lang]))
+        logger.info("   %s : %d files." % (lang, lang_count[lang]))
     output_dict = {"files_info": dict(files_info), "files_content": dict(files_content)}
-    logger.info("Saving features in {} ...".format(output))
-    with open(output, "wb") as _out:
+    logger.info("Saving features in '%s' ..." % output_path)
+    with open(output_path, "wb") as _out:
```

Comment:
same as other comments on `_in` and `_out`

Reply:
done